open_buildings/download_buildings.py

```diff
@@ -81,49 +86,53 @@ def quadkey(geojson_input):
         geojson_data = json.load(click.get_text_stream('stdin'))
     
     result = geojson_to_quadkey(geojson_data)
+    print()
     click.echo(result)
 
 @cli.command()
 @click.argument('geojson_input', type=click.File('r'), required=False)
-def WKT(geojson_input):
-    """Convert GeoJSON to Well Known Text."""
+@click.option('--geocode', type=str, is_flag=True, help='geocode using a city or region')
+def WKT(geojson_input, geocode_input):
```

Comment:
the name of the python function argument has to match the name of the click option (without --). i.e. the function definiton needs to be `def WKT(geojson_input, geocode)`